src/Monolog/LogtailHandler.php

```diff
@@ -19,27 +19,84 @@
  */
 class LogtailHandler extends BufferHandler
 {
+    const DEFAULT_BUBBLE = true;
+    const DEFAULT_BUFFER_LIMIT = 1000;
+    const DEFAULT_FLUSH_ON_OVERFLOW = true;
+    const DEFAULT_ALWAYS_FLUSH_AFTER_MILLISECONDS = 5000;
+
+    /**
+     * @var int|null $alwaysFlushAfterMs
+     */
+    private $alwaysFlushAfterMs;
```

Comment:
I know we're not big on consistency between our clients but I would call this `flushInterval` or `flushIntervalMs`. I think that's how we call similar functionality in other clients.
Wdyt? 🙏

Reply:
You're completely right there, thanks!